tests/enableCache/perspectives.js

```diff
@@ -86,7 +87,7 @@ describe(`tests/enableCache/perspectives.js, api: GET ${path} >`, () => {
         return done(err);
       }
 
-      redisCache.get('/v1/perspectives', (cacheErr, reply) => {
+      redisCache.get(md5('/v1/perspectives'), (cacheErr, reply) => {
```

Comment:
use promise style redis command e.g. getAsync instead of get, wherever you're using the callback style redis commands